src/telemTestTool.pro

```diff
@@ -2,7 +2,7 @@
     error( "Couldn't find the common.pri file!" )
 }
 
-QT += widgets
+QT += widgets serialport
```

Comment:
I remember taking it out for some reason, I think it made the travis build fail on the UCSetup Script? I'll try it again with the common in there first though.